numpy/core/tests/test_deprecations.py

```diff
@@ -741,6 +741,7 @@ def test_string(self):
         self.assert_deprecated(np.datetime64, args=('2000-01-01T00Z',))
 
     @dec.skipif(not _has_pytz, "The pytz module is not available.")
+    @dec.knownfailureif(sys.version_info[0:2] < (2, 7))
```

Comment:
Should I add some sort of explanation here? This should only be in 1.11.x, but I don't know if that is something people want or what the one line/sentence answer would be here.